### DISCUSSION TITLE
How can I force Microsoft login popup to appear in my Node.js, Express.js, and Passport.js application despite existing browser credentials?

### DISCUSSION THREAD

**Discussion by bidyut10** [jaredhanson/passport#1018] [Help]
I've implemented Microsoft authentication using Node.js, Express.js, and Passport.js in my web application. The authentication process works seamlessly. However, I noticed that if I'm already logged in using my Microsoft account in the browser, subsequent attempts to log in using Microsoft on my website skip the login popup and directly authenticate using the saved browser credentials. While this might seem convenient, I want to ensure that the Microsoft login popup always appears to maintain security and user control.

Here's how my current authentication flow works:
1. Users visit my website.
2. They click on the Microsoft login button.
3. Instead of seeing the Microsoft login popup, they are automatically logged in using their saved browser credentials if they're already logged in on Microsoft in their browser.

Here's what I want to achieve:
1. Users visit my website.
2. They click on the Microsoft login button.
3. A Microsoft login popup appears, regardless of whether they're already logged in on Microsoft in their browser or not.
4. Users enter their Microsoft credentials in the popup to authenticate.

I understand that this behavior is due to the browser handling the authentication process when it detects saved credentials for Microsoft. However, for security and user experience reasons, I want to override this behavior and always prompt the Microsoft login popup.

How can I achieve this in my Node.js, Express.js, and Passport.js application? Are there specific configurations or techniques I can implement to force the Microsoft login popup to appear every time a user attempts to log in, irrespective of their browser's saved credentials?

**Comment by adapolytech** [comment]
You can pass the prompt field (Indicates the type of user interaction that is required) on your Microsoft Passport Strategy configuration
[Docs](https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-auth-code-flow) 
Accepted value are login | none | consent | select_account
